runtime/precompiles/src/staking.rs

```diff
@@ -292,8 +310,12 @@ where
 		let min_nomination: U256 = raw_min_nomination.into();
 
 		log::trace!(target: "staking-precompile", "Result from pallet is {:?}", min_nomination);
+
 		// TODO find cost of Config associated type read
-		let gas_consumed = 0;
+		// For now assume it is as bad as a storage read in the worst case
```

Comment:
Fair enough, but Is the cost of a read of a Config associated Get type the same as a storage read? @notlesh

Reply:
It depends how `Get<_>` was implemented. If it was just `parameter_types!{...}` then it should be much faster (I would assume) but it could also be a pallet that does an actual storage read. This is what we do with our ethereum chain id.